tests/test_basic/results.py

```diff
@@ -2,7 +2,7 @@
 
 import sys
 
-sys.path.insert(0, '../../src/utils')
+sys.path.insert(0, '../..')
```

Comment:
Maybe it's just me, but it seemed more intuitive that the prevous ``sys.path.insert(...)`` call was there to allow the Python API to be imported. I would either add a comment to this mentioning the reason for ``sys.path.insert`` or move this line below the ``#import statepoint`` comment below to make it more transparent to new developers what exactly is going on.